code/Player/Grub/Controller/Mechanics/NoclipMechanic.cs

```diff
@@ -22,11 +23,13 @@ private static void Noclip()
 	protected override void OnStart()
 	{
 		Grub.Components.RemoveAny<AirMoveMechanic>();
+		Grub.Components.RemoveAny<UnstuckMechanic>();
```

Comment:
I think removing and recreating components might end up a little smelly in the future if we end up needing it in other scenarios too. I can think of an issue arising when we go to implement the jetpack. 

An idea I have is adding a Paused property to ControllerMechanic, and then adding a function to GrubController that handles pausing and unpausing. The management would be necessary in case there is more than 1 component that is removing and re-adding (or pausing and unpausing) components at a time.

It's fine for now, but it's something to think about for the future.